File: aufgabe01/nhohmann/fizzbuzz.py
import sys

def process(num):
    out = "0"
    for i in range(1, num+1):
	out += " FizzBuzz" if i%15==0 else " Fizz" if i%3==0 else " Buzz" if i%5==0 else " "+str(i)
    return out

if len(sys.argv) > 1:
    print process(int(sys.argv[1]))
else:
    print "Missing argument"

File: aufgabe01/rdiederichse/fizzbuzz.rb
# works for n > 0
if __FILE__ == $0
   f = lambda { |i| 
      return "FIZZBUZZ" if ((i % 5) + (i % 3)) == 0
      return "FIZZ" if (i % 3) == 0
      return "BUZZ" if (i % 5) == 0
      return i
   }
   1.upto(ARGV[0].to_i) { |i| puts(f.call i)}
end
